src/rut-helpers.ts

```diff
@@ -14,6 +14,9 @@ export function rutValidate(value: string) {
   }
 
   const rut: string = rutClean(value);
+
+  if (!rut.match(/\d{2}(\d|[kK])$/)) return false;
```

Comment:
I think there might be a couple of issues with this solution:
- It leaves out valid 2 digit ruts, like `1-9`. Probably 100% of the times a rut like that is inputed is not meant to be a real rut, but I'm not sure if we should include that check as part of the validation. What do you think?
- It only checks for the end, so things like `76182K851K` could still pass the validation